test/integration/custom_executable_test.rb

```diff
@@ -86,8 +88,6 @@ class CustomExecutableTest < Minitest::Test
             "expected no updates to any deprecated references files besides timeline/deprecated_references.yml")
 
           assert_match(/`deprecated_references.yml` has been updated./, captured_output)
-        ensure
```

Comment:
The entire application fixture directory is deleted during teardown.  There is no need to create Tempfiles nor ensure cleanup.